pkgs/tools/wayland/mpvpaper/default.nix

```
@@ -0,0 +1,59 @@
+{ stdenv
+, lib
+, meson
+, cmake
+, ninja
+, wlroots
+, wayland
+, wayland-protocols
+, egl-wayland
+, glew-egl
+, mpv
+, pkg-config
+, fetchFromGitHub
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mpvpaper";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "GhostNaN";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-1+noph6iXM5OSNMFQyta/ttGyZQ6F7bWDQi8W190G5E=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    wlroots
+    wayland
+    wayland-protocols
+    egl-wayland
+    glew-egl
+    mpv
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/mpvpaper \
+      --prefix PATH : ${lib.makeBinPath [ mpv ]}
+
+    install -Dm644 ../mpvpaper.man $out/share/man/man1/mpvpaper.1
```

```suggestion
    nativeBuildInputs = [ installShellFiles ];

    installManPage ../mpvpaper.man
```

You may or may not need to rename `mpvpaper.man` to `mpvpaper.1` first.

---

I knew there was a better way to do this, thanks! and thanks for the review as always.

---

I tried the suggested change here and it doesn't work, I will try to look for some examples of its usage.

---

Done! with the help of @PedroHLC